src/_guides/language/analysis-options.md

```diff
@@ -47,10 +47,8 @@ add static analysis to your tool, see the
 
 ## The analysis options file
 
-Place the analysis options file at the root of the package,
-in the same directory as the pubspec file.
-By convention, the file is called `.analysis_options`, but it
-can also be named `analysis_options.yaml` (without the leading dot).
+Place the analysis options file, `analysis_options.yaml`,
```

Comment:
We should have a note about this change, especially since `.analysis_options` files might be around for a while.

Basically help them understand (1) that they might see the . name and (2) that they should change the name if they do see it, since we expect support for it to go away.